### PR TITLE
ISSUE #3216 - You can upload files to containers that are being processed

### LINKED ISSUE #3216: You can upload files to containers that are being processed
### Description
When a container is being processed or queued, the ui allows to upload a new 

- This is happening in: prod/staging/both 
- This requires a hotfix to production: yes/no 

### Steps to replicate
- Navigate to a containers route
- Upload a new revision
- Refresh

### Current Behaviour
- It allows the user to upload a new revision straight away

### Expected Behaviour
- The interface should forbid the user to upload a new revision while is processing or queued

Like in v4:
image

### PR DESCRIPTION
**Pull request by The-Daniel**
This fixes #3216 

#### Description
- Disables/hides upload revision buttons in the containers dashboard if a container has a revision still being processed
- Disables container option in the uploads modal if the container has a revision still being processed

#### Test cases
- Upload a revision to a container
- Try to upload a new revision to this container through the containers dashboard or upload revision modal

### CODE REVIEW

**Comment by george-hadfield** [comment]
I'm not sure if I'm doing this incorrectly or not, but I was able to upload two of the same files as separate revisions to the same container.

**Comment by george-hadfield** [comment]
image

Which then displays as no revisions/files after uploading

File: frontend/src/v5/ui/routes/dashboard/projects/containers/uploadFileForm/uploadList/uploadListItem/uploadListItem.component.tsx [hunk 1]
```diff
@@ -84,15 +84,23 @@ export const UploadListItem = ({
...
+						containerDesc: value.desc,
+					};
+					Object.keys(conversion).forEach((key: any) => {
+						if (conversion[key] || key === 'containerName') {
+							setValue(key, conversion[key]);
+							updateValue(key);
+						}
 					});
 					trigger('containerName');
 				}}
```
Comment (sanmont3drepo) [inline frontend/src/v5/ui/routes/dashboard/projects/containers/uploadFileForm/uploadList/uploadListItem/uploadListItem.component.tsx:106]: Can we put this function outside the jsx code?

**Review by sanmont3drepo — CHANGES_REQUESTED** [review]

**Review by george-hadfield — DISMISSED** [review]

**Review by sanmont3drepo — APPROVED** [review]

**Pull request merged**